tutorials/neurips_challenge_quickstart.md

```diff
@@ -129,14 +129,16 @@ The following command will download and preprocess the Dolly15k dataset for the
 ```bash
 python scripts/prepare_dolly.py \
   --checkpoint_dir checkpoints/stabilityai/stablelm-base-alpha-3b \
-  --destination_path data/dolly-stablelm3b \
-  --max_seq_length 2048
+  --destination_path data/dolly-stablelm3b
 ```
 
-**Important note**
+--- 
+**Important Note**
 
 The preprocessed dataset is specific to the StableLM 3B model. If you use a different model like Falcon or Llama 2 later, you'll need to process the dataset with that model checkpoint directory. This is because each model uses a different tokenizer.
 
+--- 
```

Comment:
GitHub has this special note notation

```suggestion
> [!NOTE]
> The preprocessed dataset is specific to the StableLM 3B model. If you use a different model like Falcon or Llama 2 later, you'll need to process the dataset with that model checkpoint directory. This is because each model uses a different tokenizer.
```

Reply:
Nice, I had no idea!